app/templates/eg022_kba_authentication.html

```diff
@@ -0,0 +1,40 @@
+<!-- extend base layout --> {% extends "base.html" %}  {% block content %}
+
+<h4>22. Send an envelope using an Knowledge Based Authentication for multi-factor recipient authentication</h4>
```

Comment:
I'm not a native English speaker, but shouldn't this say "... .a Knowledge Based..." ?

Reply:
Agreed, fixed.